ui/package.json

```diff
@@ -63,6 +63,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^14.2.1",
+    "@types/node": "^20.14.10",
```

Comment:
As the UI project is currently working under Node 18, I think that the version of this dependency has to be aligned with it: `"@types/node": "^18.15.3",`

Reply:
Removed this dependency.